MeshLib/Elements/Element.h

```diff
@@ -207,19 +207,19 @@ class Element
 	 */
 	virtual bool testElementNodeOrder() const = 0;
 
+	/// Return a specific edge node.
+	virtual Node* getEdgeNode(unsigned edge_id, unsigned node_id) const = 0;
 
 protected:
 	/// Constructor for a generic mesh element without an array of mesh nodes.
 	Element(unsigned value = 0, std::size_t id = std::numeric_limits<std::size_t>::max());
 
-	/// Return a specific edge node.
-	virtual Node* getEdgeNode(unsigned edge_id, unsigned node_id) const = 0;
-
 	/// Sets the element ID.
 	virtual void setID(std::size_t id) { this->_id = id; }
 
 	Node** _nodes;
 	std::size_t _id;
+	double _content;
```

Comment:
``_geo_size_quantity``. Or give a comment as ``Length for 1D, area for 2D, and volume for 3D, respectively.``